universum/modules/output/html_output.py

```diff
@@ -11,16 +11,22 @@
 
 class HtmlOutput(BaseOutput):
 
+    default_log_name = "universum_log"
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._filename = None
+        self._log_name = self.default_log_name
+        self._log_path = None
         self.artifact_dir_ready = False
         self._log_buffer = []
         self._block_level = 0
         self.module_dir = os.path.dirname(os.path.realpath(__file__))
 
+    def set_log_name(self, log_name):
```

Comment:
Do we really need these setters? Can we maybe pass the params to init?..

Reply:
`set_log_name()` - removed
`set_artifact_dir()` - yes. `HtmlOutput` module is created under `Output` class, but only `ArtifactCollector` class is aware of the artifact directory path for the current build.